source/ssb-chart/templates/access-ingress-internal-authorization-policy.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if and .Values.access (and .Values.access.ingress .Values.access.ingress.internal) }}
+{{- range .Values.access.ingress.internal }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: "{{ template "app.name" $ }}-ingress-{{.namespace }}-{{ .application }}-auth-policy"
+  labels:
+{{ include "default.labels" $ | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "app.name" $ }}
+  action: ALLOW
+  rules:
+    {{- $fromPrincipal := printf "cluster.local/ns/%s/sa/%s-sa" .namespace .application }}
+  {{- if not .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+  {{- else }}
+    {{- range .rules }}
+    - from:
+        - source:
+            principals: [ "{{ $fromPrincipal }}" ]
+            {{- if .jwt }}
+            {{- /* The request identity is in the format of "<ISS>/<SUB>" */}}
+            requestPrincipals: [ "{{ .jwt.issuer }}/*" ]
+            {{- end}}
+{{/*      If .paths or .methods are defined*/}}
```

Comment:
I assume most of these comments are intended for internal use during development? 
However I suggest we keep comments at a minimum, and remove them before releasing. This one would probably also show regardless of the next section being included in the rendered output or not. Same with the next comment.

Reply:
Yes, this was used during development before anything was implemented. Cleaned up.